src/components/WheelPicker/index.tsx

```diff
@@ -148,7 +148,7 @@ const WheelPicker = <T extends WheelPickerItemValue>(props: WheelPickerProps<T>)
     preferredNumVisibleRows: numberOfVisibleRows
   });
 
-  const prevInitialValue = useRef(initialValue);
+  const shouldInvokeOnChange = useRef(false);
```

Comment:
It's better that the flag will represent the edge case and not the common case. 
Meaning that `false` will be the usual value, so it's better to name it something like `shouldSkipNextOnChange` or something similar. 
This way its value will be usually false and only when turning it on set it to true.

Reply:
Changed